protos/isspb/isspb.proto

```diff
@@ -43,9 +43,11 @@ message Checkpoint {
 
 message SBInstanceMessage {
   oneof type {
-    isspbftpb.Preprepare pbft_preprepare = 1;
-    isspbftpb.Prepare    pbft_prepare    = 2;
-    isspbftpb.Commit     pbft_commit     = 3;
+    isspbftpb.Preprepare        pbft_preprepare         = 1;
+    isspbftpb.Prepare           pbft_prepare            = 2;
+    isspbftpb.Commit            pbft_commit             = 3;
+    isspbftpb.SignedViewChange  pbft_signed_view_change = 4;
+    isspbftpb.PreprepareRequest pbft_preprepare_request = 5;
```

Comment:
Looks unrelated to this PR...

Reply:
It's a message required by the view change implementation, for the new primary to ask for missing preprepare messages. You are right, it's not related to this PR, it slipped in by accident.